We don't want to pull in unistd.h here for now as it causes
conflicting definitions of the syscall API. Maybe solve better
in musl at some later point.

--- a/lib/scudo/standalone/common.h
+++ b/lib/scudo/standalone/common.h
@@ -17,7 +17,9 @@
 
 #include <stddef.h>
 #include <string.h>
+#if SCUDO_ANDROID
 #include <unistd.h>
+#endif
 
 namespace scudo {
 
